src/LSM6DS3.cpp

```diff
@@ -31,6 +31,8 @@
 #define LSM6DS3_CTRL7_G            0X16
 #define LSM6DS3_CTRL8_XL           0X17
 
+#define LSM6DS3_OUT_TEMP_L 			   0X20
```

Comment:
```suggestion
#define LSM6DS3_OUT_TEMP_L         0X20
```
Use spaces for alignment. People can have true tabs set to any width so what looks aligned with your tabs setting is not aligned for someone else's tab setting.